site/src/components/site-header.md

```diff
@@ -0,0 +1,499 @@
+---
+title: Site header
+description: The Site header component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - site-header
+---
+
+# Site header
+
+Site header component allows adding full width container to hold everything site's header might need - the logo, brand link, navigation, and other controls.
+
+**This page is best viewed with the content section full screened because banners are intended to be full width elements. Go full screen by clicking the button in the top right corner.**
```

Comment:
I think this disclaimer isn't needed here. This doc looks good even on fairly narrow screens.

Reply:
Updated